core-rust/core-api-server/src/core_api/conversions/pagination.rs

```diff
@@ -0,0 +1,83 @@
+use radix_engine::types::*;
+
+use crate::core_api::*;
+
+pub struct SizeRange {
+    pub min: usize,
+    pub default: usize,
+    pub max: usize,
+}
+
+pub fn extract_valid_size(
```

Comment:
is this secretly a `size_range.resolve_valid(Option<i32>) -> Result`?

Reply:
Possibly, but would slightly prefer to keep it as-is, for conformity / discoverability.

(Currently I use `to_api_` and `extract_` autocompletes in my IDE to find relevant helper mappers and extractors.)